src/sinks/interamap/src/gps_cache.py

```
@@ -81,7 +81,7 @@ def remove_info(self, info):
 
             self.storage_update.emit((self.StorageUpdateType.REMOVE, info))
         except KeyError:
-            raise "Error: Unstored info to be removed" + str(info)
+            raise KeyError(f"Error: Unstored info to be removed {info}")
```

Ruff (0.16.2)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)